dictionaries/people-names/src/names.txt

```diff
@@ -0,0 +1,2 @@
+# List of names, one per line.
```

Comment:
maybe you could/should think about adding a footer with a comment in each dictionary saying please add your new entry here. And/or mention it in the header of each dictionary like this one

It would be part of another issue I could create if your are ok with it

Reply:
The sorting algorithm preserves comments.

The following is allowed in a source file:

```text
# Keywords
if
then

# Std Library
printf
sprintf

# Math
rand
rad
tanh
```

It will preserve each section and sort the entries.

It is possible to add a footer like:

```
Terry
Wouter
# Add new names above this line
```

But it someone adds a name after, then I need to manually fix it.

Reply:
Thanks for sharing these information.

So yes, then it should be explained in the header maybe